website/docs/docs/build/measures.md

```diff
@@ -56,8 +56,32 @@ Supported aggregations include:
 | sum_boolean       | A sum for a boolean type |
 | count_distinct    | Distinct count of values |
 | median           | Median (p50) calculation across the values |
-| percentile        | Percentile calculation across the values  |
+| percentile        | Percentile calculation across the values. |
 
+**Percentile aggregation example**<br />
+If you're using the `percentile` aggregation, you must use the `agg_params` field under it to specify details for the percentile aggregation (such as what percentile to calculate and whether to use discrete or continuous calculations).
+
+```yaml
+name: p99_transaction_value
+description: The 99th percentile transaction value
+expr: transaction_amount_usd
+agg: percentile
+agg_params:
+  percentile: .99
+  use_discrete_percentile: False  # False calculates the discrete percentile, True calculates the continuous percentile
```

Comment:
This comment is backwards - False calculates continuous, True calculates discrete.

Reply:
thanks so much for flagging this @tlento ! have updated the examples 🙏